src/detection.c

```diff
@@ -314,10 +336,25 @@ void side_sensors_calibration(void)
 	for (i = 0; i < SIDE_CALIBRATION_READINGS; i++) {
 		left_temp += distance[SENSOR_SIDE_LEFT_ID];
 		right_temp += distance[SENSOR_SIDE_RIGHT_ID];
-		sleep_ticks(SENSORS_SM_TICKS);
+		sleep_ticks(SENSORS_SM_TICKS_SENSOR);
 	}
-	calibration_factor[SENSOR_SIDE_LEFT_ID] +=
+	calibration_factor_sensor[SENSOR_SIDE_LEFT_ID] =
 	    (left_temp / SIDE_CALIBRATION_READINGS) - MIDDLE_MAZE_DISTANCE;
-	calibration_factor[SENSOR_SIDE_RIGHT_ID] +=
+	calibration_factor_sensor[SENSOR_SIDE_RIGHT_ID] =
```

Comment:
Maybe `sensor_calibration_factor`?

Anyway, do not delete the `+=`! :stuck_out_tongue_winking_eye: